float64 x_q1
float64 y_q1
float64 x_q2
float64 y_q2
float64 x_w
float64 y_w
float64 x_r
float64 y_r
float64 forward_velocity
float64 lateral_velocity


----
bool success